api/envoy/extensions/tracers/opentelemetry/resource_detectors/v3/environment_resource_detector.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package envoy.extensions.tracers.opentelemetry.resource_detectors.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.tracers.opentelemetry.resource_detectors.v3";
+option java_outer_classname = "EnvironmentResourceDetectorProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/tracers/opentelemetry/resource_detectors/v3;resource_detectorsv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Environment Resource Detector config]
+
+// Configuration for the Environment Resource detector extension.
+// The resource detector reads from the ``OTEL_RESOURCE_ATTRIBUTES``
```

Comment:
Did you consider having this information in the config instead of in an environment variable?